src/dataMap/metaManager/metaSchema.js

```diff
@@ -1019,6 +1019,8 @@ export default () => {
 
     /**
      * Class name for the Handsontable container element.
+     * It can also be used to assign class names for table cells,
+     * for example, if it is provided in `cells` or `columns` options.
```

Comment:
I think it should be more explicitly stated, e.g.:

```
Class name for the current element. The interpretation depends on the level on which this option is provided in the <a href="https://handsontable.com/docs/8.2.0/Options.html">cascading configuration</a>. If `className` is provided on the first (constructor) level, it will the applied to the Handsontable container. If `className` provided on the second (`column`) or the third (`cell` or `cells`) level, it will be applied on the table cell.
```